notebooks/en/enterprise_cookbook_argilla.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>"..., while just rating a response as "4 (good)" is ambiguous"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>